metadata-ingestion/tests/integration/sql_server/setup/setup.sql

```diff
@@ -92,3 +92,62 @@ GO
 EXEC dbo.sp_add_jobserver
     @job_name = N'Weekly Demo Data Backup'
 GO
+
+CREATE DATABASE LINEAGEDB_SLAVE;
+GO
+USE LINEAGEDB_SLAVE;
+GO
+CREATE TABLE [table_for_synonym] (id VARCHAR(MAX));
+GO
+
+CREATE DATABASE LINEAGEDB_MASTER;
+GO
+USE LINEAGEDB_MASTER;
+GO
+CREATE SCHEMA schema_with_lineage;
+GO
+CREATE TABLE [schema_with_lineage].[table_number_one] (id VARCHAR(MAX));
+GO
+CREATE TABLE [schema_with_lineage].[table_number_two] (is_updated BIT);
+GO
+CREATE VIEW [schema_with_lineage].[view_of_table_number_two] AS SELECT is_updated FROM [schema_with_lineage].[table_number_two];
+GO
+CREATE TABLE [schema_with_lineage].[table_number_three] (order_number INT, weight VARCHAR(MAX));
+GO
+CREATE SYNONYM [schema_with_lineage].[ghost_table] FOR [LINEAGEDB_SLAVE].[dbo].[table_for_synonym];
+GO
+CREATE PROCEDURE [schema_with_lineage].[procedure_number_one]
+AS
+BEGIN
+	DECLARE
+		@t1 VARCHAR(MAX)
+	SELECT @t1 = [id] FROM [schema_with_lineage].[table_number_one];
+END;
+GO       
+CREATE PROCEDURE [schema_with_lineage].[procedure_number_two]
+AS
+BEGIN
+	DECLARE @t1 INT = 1
+	UPDATE [schema_with_lineage].[view_of_table_number_two] SET [is_updated] = CASE
+                                                                                   WHEN @t1 = 1 THEN 1
+		                                                                           ELSE 0 
```

Comment:
this file seems to have a mix of tabs and spaces for indentation - was that intentional?